### PR TITLE
OKTA-698520 Add target and source terms to OAuth 2.0 Org2Org

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add references to source/target org terminology to align with other Org2Org documentation.
- **Is this PR related to a Monolith release?** No
### Resolves:

* [OKTA-698520](https://oktainc.atlassian.net/browse/OKTA-698520)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/index.md](https://github.com/okta/okta-developer-docs/blob/4f81532f73d49678288b72278f5bd25bf5357bd4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/95479ce1-9b67-4412-a9c9-1e4f9d464ad3) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md](https://github.com/okta/okta-developer-docs/blob/4f81532f73d49678288b72278f5bd25bf5357bd4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2870e174-542f-41e0-b2c2-30aa4258ecf2) | 96  | 39  | 76  | 91  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.8b9c6dff-6142-4eeb-87b2-6e1641c20407?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — COMMENTED** [review]
couple edits and comments

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/index.md](https://github.com/okta/okta-developer-docs/blob/02c6ac9e5060ebeab4a7d0c707a9788f5f5fc510/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2199c8ad-01ab-4776-9f56-8bd8a06bfb0a) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md](https://github.com/okta/okta-developer-docs/blob/02c6ac9e5060ebeab4a7d0c707a9788f5f5fc510/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/dfed85bd-16b7-4c99-bf59-126301b1bffc) | 96  | 39  | 76  | 91  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.7972c4d2-e6bb-4283-8b25-0b67d68b7dce?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/index.md](https://github.com/okta/okta-developer-docs/blob/c4edccbd7299341005aa7b2b2f674883685f3aac/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3502972d-0879-44f7-9a26-0df0b3d91315) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md](https://github.com/okta/okta-developer-docs/blob/c4edccbd7299341005aa7b2b2f674883685f3aac/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fc8b6235-6bfa-4b05-98bb-3480cb20b7ff) | 96  | 38  | 78  | 91  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.c7689e45-5641-466c-8d76-bb31acc96bd1?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+excerpt: Learn how to synchronize users and groups between orgs with OAuth 2.0 in a multi-tenant solution. Use the Okta Org2Org app in a hub-and-spoke (target-and-source) configuration with OAuth 2.0 as the authentication scheme for the provisioning connection.
 layout: Guides
 ---
 
-This guide explains how to securely configure Okta hub-and-spoke orgs to synchronize users and groups by using OAuth 2.0 in a [multi-tenant solution](/docs/concepts/multi-tenancy/). The connection between Okta orgs is configured as an OAuth 2.0 [Org2Org app integration](https://help.okta.com/okta_help.htm?type=oie&id=ext-org2org-intg). This connection provides API access to scoped data using the OAuth 2.0 Client Credential flow.
+This guide explains how to securely configure Okta hub-and-spoke orgs to synchronize users and groups by using OAuth 2.0 in a [multi-tenant solution](/docs/concepts/multi-tenancy/). The provisioning connection between Okta orgs is configured as an OAuth 2.0 [Org2Org app integration](https://help.okta.com/okta_help.htm?type=oie&id=ext-org2org-intg). This connection provides API access to scoped data using the OAuth 2.0 Client Credential flow.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md:7]:
```suggestion
This guide explains how to securely configure Okta hub-and-spoke orgs to synchronize users and groups using OAuth 2.0 in a [multi-tenant solution](/docs/concepts/multi-tenancy/). In the hub-and-spoke model, the spoke org is also referred to as the source org and the hub org is the target org. The provisioning connection between Okta orgs is configured as an OAuth 2.0 [Org2Org app integration](https://help.okta.com/okta_help.htm?type=oie&id=ext-org2org-intg). This connection provides API access to scoped data using the OAuth 2.0 Client Credential flow.
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/index.md](https://github.com/okta/okta-developer-docs/blob/6b16b395ac662d4c3b85af670fc33fc28ad54a3b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bdd73280-c9a7-46d1-ad59-75a50c6ed7a1) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md](https://github.com/okta/okta-developer-docs/blob/6b16b395ac662d4c3b85af670fc33fc28ad54a3b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bf68e13c-d0eb-4746-8535-844264833487) | 96  | 39  | 78  | 91  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.9815e11a-6cc5-4060-9884-a3004565e03d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md [hunk 2]
```diff
@@ -46,28 +45,28 @@ In this configuration:
 
 * At the spoke org level, an Org2Org app represents the OAuth 2.0 client
 
-* To limit access to the OAuth 2.0 clients, the hub-org service app needs to be granted with allowed [OAuth 2.0 scopes](https://developer.okta.com/docs/api/oauth2/#okta-admin-management).
+* To limit access to the OAuth 2.0 clients, the hub org service app needs to be granted with allowed [OAuth 2.0 scopes](https://developer.okta.com/docs/api/oauth2/#okta-admin-management).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md:48]:
```suggestion
* To limit access to the OAuth 2.0 clients, the hub-org service app needs to be granted with allowed [OAuth 2.0 scopes](https://developer.okta.com/docs/api/oauth2/#okta-admin-management).
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/index.md](https://github.com/okta/okta-developer-docs/blob/d4a5b8c67f54a1b268f015ada9dd34460677cef1/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/007ab4cf-ced9-4443-91a6-655570a963ab) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md](https://github.com/okta/okta-developer-docs/blob/d4a5b8c67f54a1b268f015ada9dd34460677cef1/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/14960e13-536a-4028-926a-68734fd921c8) | 96  | 39  | 81  | 91  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b62f9ee2-0465-4050-ad95-1a9e87e01cd3?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**